cpu/stm32f1/periph/i2c.c

```diff
@@ -209,12 +209,14 @@ int i2c_read_bytes(i2c_t dev, uint8_t address, char *data, int length)
             restoreIRQ(state);
 
             DEBUG("Wait for RXNE == 1\n");
-            while (!(i2c->SR1 & I2C_SR1_RXNE));
+            while (!(i2c->SR1 & I2C_SR1_RXNE))
```

Comment:
Okay. What do you think of instead of `while(asdf);` we change it to `while(asdf) {}`?